src/System.Net.HttpListener/src/System/Net/HttpListenerContext.Windows.cs

```diff
@@ -132,7 +132,7 @@ internal void Close()
 
                     // For unsafe connection ntlm auth we dont dispose this identity as yet since its cached
                     if ((user != null) &&
-                        (_user.Identity.AuthenticationType != AuthConstants.NTLM) &&
+                        (_user.Identity.AuthenticationType != NegotiationInfoClass.NTLM) &&
```

Comment:
Reading the code, I think the name `AuthConstants` or a new enum `AuthenticationType` would be better.
For example, in this case the check is for NTLM. Negotiate would be a different type of Auth.